test/videojs.ads.test.js

```diff
@@ -1005,13 +1005,13 @@ QUnit.test('ad impl can notify contrib-ads there is no preroll', function(assert
 
 });
 
-QUnit.test('ad impl can notify contrib-ads there is no postroll', function(assert) {
+// QUnit.test('ad impl can notify contrib-ads there is no postroll', function(assert) {
```

Comment:
For the future, `QUnit.skip` achieves this and leaves a reminder around to re-enable the test later.